# Initial plan command
string vns_conf
float64 planning_duration
Plan plan_prototype